### PR TITLE
Switching to Infinispan 11

### PR DESCRIPTION
**Pull request by ctron**
### Type of change

- Bugfix
- Refactoring

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: systemtests/src/main/java/io/enmasse/systemtest/iot/IoTUtils.java [hunk 1]
```diff
@@ -43,9 +22,33 @@
...
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
 
+import static io.enmasse.systemtest.platform.Kubernetes.getClient;
+import static io.enmasse.systemtest.platform.Kubernetes.iotConfigs;
+import static io.enmasse.systemtest.platform.Kubernetes.iotTenants;
+import static io.enmasse.systemtest.time.TimeMeasuringSystem.Operation.startOperation;
+import static io.enmasse.systemtest.utils.TestUtils.TimeoutHandler.explain;
+import static java.util.Optional.ofNullable;
+import static org.assertj.core.api.Assertions.assertThat;
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/iot/IoTUtils.java:45]: we use org.hamcrest.MatcherAssert.assertThat; in other tests so I think you iot should use it as well

Comment (ctron) [reply systemtests/src/main/java/io/enmasse/systemtest/iot/IoTUtils.java:45]: It is not about the `assertThat`, that just enabled the use of "assertj", which makes a few tests easier later on.

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot testcase=io.enmasse.systemtest.iot.isolated.http.HttpAdapterTest

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot testcase=io.enmasse.systemtest.iot.isolated.http.HttpAdapterTest

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot testcase=io.enmasse.systemtest.iot.isolated.http.HttpAdapterTest

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
@enmasse-ci run tests ocp4 profile=iot

**Comment by ctron** [comment]
rebased

**Pull request merged**